packages/core/src/external/carequality/ihe-gateway-v2/utils.ts

```diff
@@ -17,9 +18,16 @@ export function timestampToSoapBody(createdTimestamp: string): string {
   return dayjs(createdTimestamp).toISOString();
 }
 
-export function toArray<T>(input: T | T[]): T[] {
-  if (input == undefined) {
+export function toArray<T>(input: T | T[] | "" | undefined): T[] {
```

Comment:
we have this function on the shared package. Can we make this update there and mark this one here as deprecated plz?

Reply:
I just removed this one and changed eveerything to use the updated one in shared